index.html

```diff
@@ -288,6 +288,48 @@ <h5>Cups of coffee</h5>
   </div>
  <!--"Counter" ends here-->
 
+ <!-- Gallery starts here -->
+  <div class="gallery">
+
+    <div class="container">
+
+      <h1>Gallery</h1>
+
+      <!-- The Carousel -->
+      
+      <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
+        <ol class="carousel-indicators">
+          <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+          <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+          <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+        </ol>
+        <div class="carousel-inner">
+          <div class="carousel-item active">
+            <img src="assets/images/group_Photos/DSC_0733.jpg" class="d-block w-100" alt="IMG 1">
```

Comment:
@KondaHemanth The image `alt` tag matters a lot in SEO, please use the keyword here not **IMG 1**, **PHOTO** etc.

Reply:
@Manasvi070902 @KondaHemanth 
Use suitable alt tags and image names. Rename the images to something meaningful as well. Do this in all pages.

@vinitshahdeo thank you for taking out time and helping review ✌🏻